test/modeling/test_dpr.py

```diff
@@ -31,10 +32,10 @@ def test_dpr_modules(caplog=None):
     devices, n_gpu = initialize_device_settings(use_cuda=True)
 
     # 1.Create question and passage tokenizers
-    query_tokenizer = get_tokenizer(
+    query_tokenizer = AutoTokenizer.from_pretrained(
```

Comment:
Why are we using `AutoTokenizer` here instead of the `FeatureExtractor`? As far as I can see, you replaced some instances of `get_tokenizer` in the tests by `AutoTokenizer` and some by `FeatureExtractor`. Could you maybe explain when to use whiche one of these two?

Reply:
Yes: these tests are tightly couples with `AutoTokenizer` specifically. They access many of its internal attributes, which are normally not exposed by `FeatureExtractor`. So rather than exposing them or change the access path all around in the test, I decided to instantiate an `AutoTokenize`r directly instead of going through the additional hoop of `get_tokenizer`. This tests in particular are not testing any feature of `get_tokenizer` anyway.